Python/cp/Cyclically_rotate.py

```diff
@@ -0,0 +1,105 @@
+'''
+To cyclically (clockwise) rotate a list of integers by N
+(N is user defined)
+'''
+
+
+def Rotate(list, N):
+    for i in range(N):
+        a = list.pop()
+        list.insert(0, a)
+    return list
+
+try:
+    list = []
+
+    # For the list rotation to be dynamic it has been put in a loop
+    # While the loop is true the user gets 4 options
+
+    while True:
+        op = int(input('''
+Press-->
+1 to insert element in the list
+2 to rotate list cyclically
+3 to display list
+4 to exit
+'''))
+
+        if op == 1:
+            ele = int(input("enter element to put in list "))
```

Comment:
It would be better if user could input all the elements in the list at once. You can ask user to enter the number of elements in the list first and then take the elements accordingly.

Reply:
Alright